admin/src/pages/CreationConfirm.vue

```diff
@@ -70,6 +70,10 @@ export default {
           .catch((error) => {
             this.$toasted.error(error.message)
           })
+      } else {
```

Comment:
I guess this is the case `confirmed`. Can we have a `switch` here that throws an error when no case matches?

Reply:
Yeah this would be a good alternativ